R/utils.R

```diff
@@ -27,3 +27,18 @@ add_newlines <- function(n) {
 add_spaces <- function(n) {
   rep_char(" ", n)
 }
+
+#' Combine child and internal child
```

Comment:
This looks like a function with a very narrow use case. As such I'd prefer if it was located right below the function that calls it.